standard-covoiturage_openapi.yaml

```diff
@@ -477,30 +892,131 @@ components:
         brand:
           type: string
           description: Brand of the car.
-
-    DeepLink:
+                                
+    ConnectionMessage:
       type: object
-      description: Platform specific deep-link configurations.
+      required:
+        - driverJourneyId
+        - passengerJourneyId
       properties:
-        android:
-          type: object
-          properties:
-            uri:
-              type: string
-              description: |-
-                URI compliant with Android conventions to open the webservice
-                provider mobile app on the screen presenting the specific journey
-                (see [this guide](https://blog.branch.io/technical-guide-to-deep-linking-on-android-chrome-intents/) for more details).
-            storeUrl:
-              type: string
-              description: |-
-                URL of the webservice provider mobile app on the PlayStore
-                in case the app is not yet installed on the device.
-        ios:
-          type: object
-          properties:
-            universalLink:
-              type: string
-              description: |-
-                URI compliant with iOS conventions to open the webservice
-                provider mobile app on the screen presenting the specific journey.
+        driverJourneyId:
+          type: string
+          minLength: 1
+          maxLength: 255
+          description: Driver Journey's id. It MUST be unique for a given operator.
+        passengerJourneyId:
+          type: string
+          minLength: 1
+          maxLength: 255
+          description: Passenger Journey's id. It MUST be unique for a given operator.
+        message:
+          type: string
+          maxLength: 500
+          description: Free text content of a message. The message can contain all the details (phone number, email, etc.) allowing the recipient to call back the sender in order to carpool with him/her.
+        driverOperator:
+          type: string
+          description: Name of the operator owning the DriverJourney.
+        driverOperatorWebUrl:
+          type: string
+          description: Url on the website of the operator owning the DriverJourney.
+        passengerOperator:
+          type: string
+          description: Name of the operator owning the PassengerJourney.
+        passengerOperatorWebUrl:
+          type: string
+          description: Url on the website of the operator owning the PassengerJourney.
+          
+    Booking:
+      allOf:
+        - $ref: '#/components/schemas/ConnectionMessage'
+      type: object
+      required:
+        - id
+        - status
+        - price
```

Comment:
It seems at least `passengerPickup[Lat|Lng|Date]` and `passengerDrop[Lat|Lng]` should be mandatory.